### PR TITLE
Build IMU Program

### LINKED ISSUE #3: Create IMU Class and Add IMU Madgwick Filter Algorithm

### PR DESCRIPTION
**Pull request by maroqijalil**
Closes #3

### CODE REVIEW

File: include/kansei/imu_filter.hpp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#include <iostream>
+#include <cmath>
+
+namespace kansei
+{
+
+class ImuFilter
+{
+public:
+	ImuFilter();
```
Comment (threeal) [inline include/kansei/imu_filter.hpp:36]: Use spaces instead of tabs.

File: include/kansei/imu_filter.hpp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+private:
+	// **** paramaters
+	double gain_;						 // algorithm gain
+	double zeta_;						 // gyro drift bias gain
+	WorldFrame world_frame_; // NWU, ENU, NED
+
+	// **** state variables
+	double q0, q1, q2, q3;		 // quaternion
+	float w_bx_, w_by_, w_bz_; //
```
Comment (threeal) [inline include/kansei/imu_filter.hpp:47]: Reformat comments, and remove unused ones.

**Review by threeal — CHANGES_REQUESTED** [review]
See individual comments for the requested changes.

**Review by threeal — APPROVED** [review]
Looks good to me.

**Review by finesaaa — APPROVED** [review]
Good 👌

**Pull request merged**